packages/core/lib/asciidoctor/js/asciidoctor_ext/node/template.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
```

Comment:
@mojavelinux The code is very close to the [original implementation in Ruby](https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor/converter/template.rb). To avoid code duplication, we could extract some parts and add conditional blocks or we could create a `TiltTemplateConverter` with the Tilt related bits.